src/lib/containers/entity_finder/details/view/DetailsViewTableRenderers.tsx

```diff
@@ -211,3 +387,56 @@ export const VersionRenderer = ({
     </div>
   )
 }
+
+export function CheckboxRenderer<
+  T extends {
+    isSelected: boolean
+    isDisabled?: boolean
+    setSelected: (newValue: boolean) => void
+  },
+>(props: CellRendererProps<T>) {
+  const { isSelected, isDisabled, setSelected } = props.rowData
+  return (
+    !isDisabled && (
+      <Checkbox
+        disabled={isDisabled}
+        label=""
+        checked={isSelected}
+        onChange={value => {
+          setSelected(value)
+        }}
+      />
+    )
+  )
+}
+
+export const EntityErrorRenderer = (props: EntityIdAndVersionRendererProps) => {
```

Comment:
https://user-images.githubusercontent.com/17580037/143080313-90211185-9677-4424-8d4d-b0c647c3739c.mov

Reply:
Recommend setting the skeleton animation to false if there's an error (it looks like it's loading when animated).